src/app/(application)/mortgage/_form/personal-details-form.tsx

```diff
@@ -46,7 +43,10 @@ const formSchema = z.object({
   }),
 })
 
-const PersonalDetailsForm = () => {
+interface Props {
+  onSubmitForm: (values: z.infer<typeof formSchema>) => void
```

Comment:
Let's keep it consistent. Noticed you used `onSubmit` for other forms. Let's pick any one. 

my vote: onSubmit instead of onSubmitForm